build.gradle.kts

```diff
@@ -29,18 +29,25 @@ repositories {
     maven("https://plugins.gradle.org/m2")
 }
 
+configurations.all {
+    // Dependencies provided by IntelliJ runtime
+    exclude("com.google.code.gson", "gson")
+    exclude("org.slf4j", "slf4j-api")
+}
+
 dependencies {
-    implementation("io.pebbletemplates:pebble:3.1.5")
-    implementation("org.junit.jupiter:junit-jupiter:5.8.1")
-    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.10.0-RC1")
+    implementation("com.github.doyaaaaaken:kotlin-csv-jvm:1.6.0") {
+        // Not using any of the async functionality
+        exclude("org.jetbrains.kotlinx", "kotlinx-coroutines-core")
+    }
+    implementation("com.google.code.gson:gson:2.10")
     implementation("com.launchdarkly:api-client:3.10.0")
     implementation("com.launchdarkly:launchdarkly-java-server-sdk:5.+")
-    implementation("com.google.code.gson:gson:2.7")
-    implementation("com.googlecode.json-simple", "json-simple", "1.1.1")
-    implementation("com.github.doyaaaaaken:kotlin-csv-jvm:0.14.0")
-    runtimeOnly("org.jetbrains.kotlin:kotlin-reflect:1.4.20-M2")
-    implementation("org.apache.logging.log4j", "log4j-slf4j-impl", "2.17.1")
```

Comment:
Not sure why `kotlin-reflect` or log4j were explicitly included. They are not pulled in by any other dependencies, and the plugin seems to be working fine without them.

Removing these saved almost 5Mb.